iterations_runners/iterations_runner.c

```diff
@@ -76,9 +78,16 @@ main(int argc, char **argv)
         goto clean;
     }
 
-    /* Building a JSON list */
-    fprintf(stdout, "[");
-    fflush(stdout);
+    if ((krun_iter_times =
+            calloc(krun_total_iters, sizeof(double))) == NULL) {
+        errx(EXIT_FAILURE, "%s", strerror(errno));
+        goto clean;
+    }
+
+    for (krun_iter_num = 0;
+        krun_iter_num < krun_total_iters; krun_iter_num++) {
```

Comment:
I don't even understand why it's broken onto 2 lines -- the whole thing is less than 80 chars on 1 line I think? And, if it does need breaking, surely it's better to break the minimum amount onto the second line, not the maximum?